api/openapi_server/controllers/auth_controller.py

```diff
@@ -65,6 +66,8 @@ def get_token_auth_header():
     return token
 
 def sign_up(body: dict, role: UserRole):
+    from openapi_server.controllers.admin_controller import remove_user
```

Comment:
Question: Is importing inside a function a common practice in Python? It feels odd, but that might be from my lack of Python experience and primarily using JS.